integration/hive/src/main/java/org/apache/carbondata/hive/MapredCarbonInputFormat.java

```diff
@@ -94,8 +95,9 @@ private static void populateCarbonTable(Configuration configuration, String path
       // persisted in the schema
       CarbonTable carbonTable;
       AbsoluteTableIdentifier absoluteTableIdentifier = AbsoluteTableIdentifier
-          .from(validInputPath, getDatabaseName(configuration), getTableName(configuration));
-      String schemaPath = CarbonTablePath.getSchemaFilePath(validInputPath);
+          .from(configuration.get(hive_metastoreConstants.META_TABLE_LOCATION),
+              getDatabaseName(configuration), getTableName(configuration));
+      String schemaPath = CarbonTablePath.getSchemaFilePath(absoluteTableIdentifier.getTablePath());
```

Comment:
pass the configuration also to `getSchemaFilePath` method

Reply:
done

Reply:
done